.gitignore

```diff
@@ -66,7 +66,11 @@ target/
 .generated/
 
 # MindMeld Data
-data/
+mindmeld/converter/test_run.py
```

Comment:
You shouldn't check in these files

Reply:
I think these changes came from Alans branch, I'll revert back the changes.

Reply:
my bad, I'll keep that in mind, thank you barret